pkg/controller/ready/ready_test.go

```diff
@@ -22,11 +22,11 @@ var _ = Describe("controller ready helper", func() {
 	Context("setting and unsetting ready file", func() {
```

Comment:
This file has only formatting changes. May you remove it from your PR?

Reply:
It's changed by `go fmt`, do we need to comply with this rule? Or you mean it should be address by a separate PR? Thanks.